recipes/hash-library/all/test_package/CMakeLists.txt

```
@@ -1,9 +1,6 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```
Keep 3.1 you can run `conan test` without `conan create`